src/main/resources/crafter/studio/upgrade/pipelines.yaml

```diff
@@ -121,6 +121,9 @@ pipelines:
     - currentVersion: 3.0.0
```

Comment:
What if someone is flowing through 3.0.15 or 3.0.17

Reply:
Since sites < 3.1.0 don't have a `studio_version.xml`, the upgrader assumes a `3.0.0` version. It's ok since even if it's 3.0.0, 3.0.10, 3.0.15, etc, the file will be added.